secondary_bulk_insert_script.sql

```diff
@@ -148,3 +148,38 @@ VALUES
 (21, 'Hillbrow  Hospital','012 3456 444')
 GO
 
+/*
+* Incident Report Junction Table
+*/
+
+INSERT INTO [dbo].[Incident_Report]
+           ([Incident_ID]
+           ,[Incident_Type_ID])
+
```

Comment:
There is just a space between the values statement and the other statements. Which is not there on the other ones.